src/app/utils.py

```diff
@@ -432,17 +581,18 @@ def get_license_data(issues):
     licenseIds = []
     licenseTexts = []
     for issue in issues:
-        if issue.get('pull_request') is None:
-            licenseInfo = issue.get('body')
-            if '[SPDX-Online-Tools]' in issue.get('title'):
-                licenseIdentifier = re.search(r'(?im)short identifier:\s([a-zA-Z0-9|.|-]+)', licenseInfo).group(1)
-                licenseIds.append(licenseIdentifier)
-                try:
-                    licenseXml = str(LicenseRequest.objects.get(shortIdentifier=licenseIdentifier).xml)
-                    licenseText = parseXmlString(licenseXml)['text']
-                    licenseTexts.append(clean(licenseText))
-                except LicenseRequest.DoesNotExist:
-                    pass
+        if not isinstance(issue, unicode):
+            if issue.get('pull_request') is None:
```

Comment:
In general you would rather write the clearer: `if not issue.get('pull_request')`: